ui/src/filters/filters.js

```diff
@@ -1,6 +1,36 @@
 import Vue from "vue"
 
 var Filters = {
+    bitFormat: function (size) {
+        var result;
+
+        switch (true) {
+            case size === null || size === "" || isNaN(size):
+                result = "-";
+                break;
+
+            case size >= 0 && size < 1024:
+                result = size + " bit";
+                break;
+
+            case size >= 1024 && size < Math.pow(1024, 2):
+                result = Math.round(size / 1024) + " Kbit";
```

Comment:
Should we keep the decimal figures of the value? E.g. **235,48 Mbit** is more informative than **235 Mbit**. This is especially true for bigger values using Gbit.
```suggestion
                result = Math.round(size / 1024 * 100) / 100 + " Kbit";
```
The same applies for Kbit, Mbit, Gbit and Tbit

Reply:
![Screenshot - 2021-05-19T153529 530](https://user-images.githubusercontent.com/3164851/118821787-deb71b00-b8b7-11eb-8f08-5ca30df62ed9.png)

Yes more accurate